src/controllers/authenticationController.ts

```diff
@@ -175,6 +190,17 @@ export default class AuthenticationController {
       );
     }
 
+    // LV functionality is enable only if Lollipop is enabled.
+    // With FF set to BETA or CANARY, only whitelisted CF can use the LV functionality (the token TTL is reduced if login type is `LV`).
```

Comment:
This seems to be wrong. If the FF is set to `CANARY` the LV is disabled for all the users on the current implementation:
```typescrypt
export const getIsUserElegibleForfastLogin = (
  betaTesters: ReadonlyArray<FiscalCode>,
  FF_FastLogin: FeatureFlag
) =>
  getIsUserEligibleForNewFeature<FiscalCode>(
    (fiscalCode) => betaTesters.includes(fiscalCode),
    (_fiscalCode) => false, // <----- CANARY
    FF_FastLogin
  );
```

Reply:
Nope my bad, in the inner implementation canary includes betas user, sorry. But is not very clear is an hidden behaviour
```
case "CANARY":
        return isUserCanary(i) || isUserBeta(i);
```